src/Models/BaseModel.vala

```diff
@@ -14,10 +14,10 @@
 * You should have received a copy of the GNU General Public License
 * along with Akira. If not, see <https://www.gnu.org/licenses/>.
 *
-* Authored by: Alessandro "alecaddd" Castellani <castellani.ale@gmail.com>
```

Comment:
Don't delete the previous contributors even if you renamed the file.

Reply:
I think I done it for accident, I don't recall how I deleted that line!

Btw, this file is not going to survive the next PR, since it is no more needed :) 

It's still there only because I didn't want to overload this PR, but since we changed the `ListModel`, both the `ItemModel` and the `BaseModel` are going to be dropped!